src/pages/page-teste.tsx

```diff
@@ -1,9 +1,11 @@
 import React from "react";
 
-export const HelloWorld = () => {
+import { ListUsers } from "../components/mol.list-users/list-user.component";
+
+export const PageUser = () => {
```

Comment:
Vamos parar de errar essa nomenclatura.
Em inglês, a ordem é `UserPage`, e a ordem importa porque é a ordem que estabelece a hierarquia entre os termos.  Primeiro colocamos o complemento ("do usuário", depois o sujeito ("página"). 

```suggestion
export const UserPage= () => {
```

Reply:
corrigido e não irá mais acontecer!